regulations/templates/regulations/node.html

```diff
@@ -14,6 +14,20 @@
 {% endif %}
 </p>
 {% endif %}
+
+<div class="comment-wrapper" data-section="{{ node.markup_id }}">
+  <div class="toggle">(click to toggle comment box)</div>
+  <div class="comment">
+    <h3>Enter your comment below:</h3>
+    <form>
+      <textarea role="textbox" aria-multiline="true"></textarea>
+      <div class="queued"></div>
+      <input type="file">
+      <button type="submit">Submit</button>
+    </form>
+  </div>
+</div>
```

Comment:
Hmm, this will impact _all_ nodes (including those of regulations). We can get around that by [providing a custom template for a specific type of node](https://github.com/18F/regulations-site/tree/master/regulations/templates/regulations/tree), or perhaps adding a more generic solution for "tail" data (including node.interp, below and footnotes).

Also, something we struggled with before: is a comment on just the intro text or on the whole node (including sub paragraphs)? If the latter, the box should probably be _after_ the child paragraphs.